js/components/videoQualityPanel.js

```diff
@@ -58,13 +59,16 @@ var VideoQualityPanel = React.createClass({
         'oo-selected': this.state.selected == availableBitrates[i].id
       });
 
-      if (availableBitrates[i].id == 'auto'){
+      if (availableBitrates[i].id == 'auto') {
         this.addAutoButton(bitrateButtons);
       }
+      else if (typeof availableBitrates[i].bitrate === "number") {
```

Comment:
bitrate type check should not be part of the ID check. ID can be auto and bitrate and be string. Shoud be -

```
      if (availableBitrates[i].id == 'auto') {
        this.addAutoButton(bitrateButtons);
      }
      else {
        if (typeof availableBitrates[i].bitrate === "number") {
          label = Math.round(availableBitrates[i].bitrate/1000) + ' kbps';
        } 
        else {
          label = availableBitrates[i].bitrate;
        }
        bitrateButtons.push(<li key={i}><a className={qualityBtn} key={i} onClick={this.handleVideoQualityClick.bind(this, availableBitrates[i].id)}>{label}</a></li>);
      }
```